### PR TITLE
Ingredients

### PR DESCRIPTION
**Pull request by thekarinaten**

### CODE REVIEW

File: assets/css/style.css [hunk 1]
```diff
@@ -3,40 +3,78 @@
...
+  transform: translateX(15%);
+}
+
+.about-food {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  max-width: 430px;
+  text-align: center;
```
Comment (katsu223) [inline assets/css/style.css:21]: I think it's redundant

File: assets/css/style.css [hunk 2]
```diff
@@ -3,40 +3,78 @@
...
+}
+
+.about-title {
+  color: #cc9900;
```
Comment (katsu223) [inline assets/css/style.css]:
```suggestion
  color: #c90;
```

Comment (katsu223) [reply assets/css/style.css]: Here and below

File: assets/css/style.css [hunk 3]
```diff
@@ -3,40 +3,78 @@
...
+body,
+html,
+ul,
+ol,
+li,
+p,
+h1,
+h2,
+h3,
+h4 {
```
Comment (katsu223) [inline assets/css/style.css]: U can leave it in one line

File: assets/css/style.css [hunk 4]
```diff
@@ -3,40 +3,78 @@
...
+  box-sizing: border-box;
 }
 
 .button {
   display: flex;
   justify-content: center;
   max-width: 486px;
   margin: 0 auto;
-  padding-bottom: 130px;
+  box-sizing: border-box;
```
Comment (katsu223) [inline assets/css/style.css]: You have it here - https://github.com/thekarinaten/lambda-restaurant/pull/5/files#diff-b8392010d5f6b5c8a2f493b3a11e3109R52

File: assets/css/style.css [hunk 5]
```diff
@@ -3,40 +3,78 @@
...
 }
 
 .button-black {
-  background-color: rgba(0, 0, 0, .7) ;
+  background-color: rgba(0, 0, 0, 0.7);
```
Comment (katsu223) [inline assets/css/style.css]: ?
```suggestion
  background-color: rgba(0, 0, 0, .7);
```

File: assets/css/style.css [hunk 6]
```diff
@@ -3,40 +3,78 @@
...
   border: 2px solid #000;
   justify-content: center;
   margin: 17px 15px;
   padding: 20px 50px;
 }
 
 .button-white {
-  background-color: rgba(255, 255, 255, .28);
+  background-color: rgba(255, 255, 255, 0.28);
```
Comment (katsu223) [inline assets/css/style.css]: same here

File: assets/css/style.css [hunk 7]
```diff
@@ -49,8 +87,24 @@ button {
   white-space: nowrap;
 }
 
-.promo-section {
-  padding-bottom: 130px;
+.content-title {
+  color: rgb(255, 255, 255);
```
Comment (katsu223) [inline assets/css/style.css]: ?
#fff

File: assets/css/style.css [hunk 8]
```diff
@@ -49,8 +87,24 @@ button {
...
+  font-family: 'Yeseva One', 'Arial', 'Times New Roman', serif;
+  font-size: 90px;
+  font-weight: 100;
+  max-width: 970px;
+  text-align: center;
+  margin: 0px auto;
```
Comment (katsu223) [inline assets/css/style.css]:
```suggestion
  margin: 0 auto;
```

File: assets/css/style.css [hunk 9]
```diff
@@ -49,8 +87,24 @@ button {
...
+}
+
+div {
+  box-sizing: border-box;
+}
```
Comment (katsu223) [inline assets/css/style.css]: It is defined here - https://github.com/thekarinaten/lambda-restaurant/pull/5/files#diff-b8392010d5f6b5c8a2f493b3a11e3109R54

File: assets/css/style.css [hunk 10]
```diff
@@ -49,8 +87,24 @@ button {
...
+
+header {
```
Comment (katsu223) [inline assets/css/style.css]: use .class not tag selector

File: assets/css/style.css [hunk 11]
```diff
@@ -74,6 +128,53 @@ button {
   background-repeat: no-repeat;
 }
 
+.ingredients-img {
+  padding-top: 70px;
+}
+
+.ingredients-item-2 {
+margin: 0px 30px 0px 30px
```
Comment (katsu223) [inline assets/css/style.css]:
```suggestion
  margin: 0 30px
```

File: assets/css/style.css [hunk 12]
```diff
@@ -74,6 +128,53 @@ button {
   background-repeat: no-repeat;
 }
 
+.ingredients-img {
+  padding-top: 70px;
+}
+
+.ingredients-item-2 {
```
Comment (katsu223) [inline assets/css/style.css]: Weird name. Too specific

File: assets/css/style.css [hunk 13]
```diff
@@ -74,6 +128,53 @@ button {
...
+margin: 0px 30px 0px 30px
+}
+
+.ingredients-divider {
+  padding: 25px 0px 25px 0px;
```
Comment (katsu223) [inline assets/css/style.css]:
```suggestion
  padding: 25px 0;
```

File: assets/css/style.css [hunk 14]
```diff
@@ -74,6 +128,53 @@ button {
...
+  color: #fff;
+  font-family: 'Lato', 'Arial', 'Times New Roman', sans-serif;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 24px;
+}
+
+.ingredients-wrapper {
+  align-items: center;;
+  background-color: rgba(0, 0, 0, 0.67);
```
Comment (katsu223) [inline assets/css/style.css]:
```suggestion
  background-color: rgba(0, 0, 0, .7);
```

File: assets/css/style.css [hunk 15]
```diff
@@ -74,6 +128,53 @@ button {
...
+.ingredients-txt {
+  color: #fff;
+  font-family: 'Lato', 'Arial', 'Times New Roman', sans-serif;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 24px;
+}
+
+.ingredients-wrapper {
+  align-items: center;;
```
Comment (katsu223) [inline assets/css/style.css]:
```suggestion
  align-items: center;
```

File: assets/css/style.css [hunk 16]
```diff
@@ -74,6 +128,53 @@ button {
...
+  background-color: rgba(0, 0, 0, 0.67);
+  display: flex;
+  flex-direction: column;
+  max-width: 570px;
+  text-align: center;
+  padding: 82px 70px 66px 70px;
+  margin: 92px 215px 101px 0px;
```
Comment (katsu223) [inline assets/css/style.css:159]: A u sure in such pixel perfect?

File: index.html [hunk 17]
```diff
@@ -5,10 +5,10 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css" />
-    <link rel="stylesheet" href="assets/css/style.css" />
+    <link rel="stylesheet" href="/assets/css/style.css">
```
Comment (katsu223) [inline index.html:8]: For what?

File: index.html [hunk 18]
```diff
@@ -5,10 +5,10 @@
...
     <title>Lambda Restaurant</title>
   </head>
-  <body>
```
Comment (katsu223) [inline index.html:11]: ?!?!

**Review by katsu223 — CHANGES_REQUESTED** [review]

File: assets/css/style.css (hunk 1 above)
Comment (thekarinaten) [reply assets/css/style.css:21]: idk how to place it the other way

File: assets/css/style.css (hunk 16 above)
Comment (thekarinaten) [reply assets/css/style.css:159]: should i change it?
like 80px 70px 65px 70px
90px 200px 100px 0px

File: assets/css/style.css (hunk 1 above)
Comment (katsu223) [reply assets/css/style.css:21]: I need to see the deploy to help you

File: assets/css/style.css (hunk 16 above)
Comment (katsu223) [reply assets/css/style.css:159]: I need to see the deploy, but your suggestion is better then selected code

File: assets/css/style.css (hunk 8 above)
Comment (katsu223) [reply assets/css/style.css]: not fixed

File: assets/css/style.css [hunk 19]
```diff
@@ -132,12 +113,12 @@ header {
   padding-top: 70px;
 }
 
-.ingredients-item-2 {
-margin: 0px 30px 0px 30px
+.ingredients-item-spice {
+margin: 0px 30px;
```
Comment (katsu223) [inline assets/css/style.css:120]:
```suggestion
  margin: 0px 30px;
```

File: assets/css/style.css [hunk 20]
```diff
@@ -132,12 +113,12 @@ header {
...
 }
 
 .ingredients-divider {
-  padding: 25px 0px 25px 0px;
+  padding: 25px 0px;
```
Comment (katsu223) [inline assets/css/style.css:124]:
```suggestion
  padding: 25px 0;
```
and better to use 
padding-top
padding-bottom
for such cases

**Review by katsu223 — CHANGES_REQUESTED** [review]

File: assets/css/style.css [hunk 21]
```diff
@@ -109,18 +210,16 @@ button {
...
+  margin: 0 auto;
   text-align: center;
 }
 
 .wrapper {
   max-width: 1600px;
   margin: 0 auto;
-}
-
+}
```
Comment (katsu223) [inline assets/css/style.css:223]:
```suggestion
}

```
always add blank line in the end of the file

**Review by katsu223 — CHANGES_REQUESTED** [review]

File: assets/css/style.css (hunk 1 above)
Comment (thekarinaten) [reply assets/css/style.css:21]: http://lambda.ingredients-section.surge.sh/

**Pull request merged**